stock_prices/stock_prices.py

```diff
@@ -2,14 +2,30 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    prices = iter(prices)
+    least = next(prices)
+    yield 0
+    for price in prices:
+        least = min(least, price)
+        yield price - least
+
+
```

Comment:
Hey Riley , have you tried what I suggested yesterday with restarting this approach? Remember to reach out if you need help!